dev/source/docs/building-the-code.rst

```diff
@@ -15,18 +15,42 @@ Setting up the Build Environment
 - :ref:`Setup the Build Environment on Windows <building-setup-windows>`
 - :ref:`Setup the Build Environment on MacOSX <building-setup-mac>`
 
+
 Building / Compiling
 --------------------
 
-**Linux / MacOSX users:**
+Once the build environment is set up as above, run the following commmands prior to the initial build. (For Windows users, this is using WSL, as set up above):
 
-- Linux and MacOSX users should build with waf as described in `BUILD.md <https://github.com/ArduPilot/ardupilot/blob/master/BUILD.md>`__.
+- `git clone https://github.com/ArduPilot/ardupilot`
+- `cd ardupilot`
+- `git submodule update --recursive --init`
+- `./Tools/gittools/submodule-sync.sh`
+- `./waf configure --board *BoardName*`
+- `./waf clean` (Optional; may be useful in some cases)
 
-.. youtube:: lNSvAPZOM_o
+`*BoardName*` above is the name of the board, as labeled by its associated folder in
+`/ardupiot/libraries/AP_HAL_ChibiOS/hwdef`.
```

Comment:
no, board name isn't restricted to Chibios boards.
`./waf list_boards` show all the boards